### PR TITLE
feat(app): getBranchName() to get full name

### LINKED ISSUE #1: Add new function to return complete branch name
add a new function that returns the complete branch name, keeping the old function that returns just the branch code as it is.

### PR DESCRIPTION
**Pull request by tanay13**
fixes #1

### CODE REVIEW

**Review by YashKumarVerma — CHANGES_REQUESTED** [review]
please add a new function something like `getBranchName()` because the `getBranch()` function returns just branch codes like BCE, BBS etc.


also, would recommend using a switch-case

**Comment by tanay13** [comment]
@YashKumarVerma please check now

File: src/app.ts [hunk 1]
```diff
@@ -33,31 +34,45 @@ class VITRegNumber {
...
+            this.branchName = "Electrical and Electronics engineering";
+            break;
+          case 'BIT':
+            this.branchName = "Information Technology";
+            break;
+          case 'BAI':
+            this.branchName = "Computer Science Engineering with Specialization in Artificial Intelligence and Machine Learning";
+            break;
+          default:
+            this.branchName = this.branch;
```
Comment (YashKumarVerma) [inline src/app.ts:73]: this code block should be in the function body of getBranchName()

**Review by YashKumarVerma — CHANGES_REQUESTED** [review]

File: src/app.ts (hunk 1 above)
Comment (tanay13) [reply src/app.ts]: Done! please check again

**Review by YashKumarVerma — APPROVED** [review]
looks good !

**Comment by YashKumarVerma** [comment]
thanks!

**Pull request merged**